### ISSUE TITLE
read_replicas name always has <primary_name>-replica in it

### ISSUE THREAD

**Issue by wshao12** [terraform-google-modules/terraform-google-sql-db#384]
### TL;DR

I want to create a read_replica with its own name. If I set name = "", it gives <primary_name>-replica, if I set name ="test", I got <primary_name>-replicatest. but I actually want name as "test".  Is this a bug? How I can choose my own name without "-replica" in it? 

### Expected behavior

 + resource "google_sql_database_instance" "replicas" {
      + master_instance_name           = "article"
      + name                           = "article-replicareplica_test_name"

### Observed behavior

 + resource "google_sql_database_instance" "replicas" {
      + master_instance_name           = "article"
      + name                           = "replica_test_name"

### Terraform Configuration

```hcl
terraform {
  required_providers {
    google = {
      source  = "hashicorp/google"
      version = "~> 4.44"
    }

  }
}
```

### Terraform Version

```sh
Terraform v1.3.1
on darwin_amd64
```

**Comment by wshao12** [comment]
required_providers {
    google = {
      source  = "hashicorp/google"
      version = "~> 4.44"
    }
}
